api/resolvers/search.js

```diff
@@ -279,20 +292,51 @@ export default {
             fields: ['title', 'text']
           }
         })
+
+        // force the search to include the quoted phrase
+        filters.push({
+          multi_match: {
+            query: quote,
+            type: 'phrase',
+            fields: ['title', 'text']
+          }
+        })
       }
 
       // query for search terms
       if (query.length) {
         // keyword based subquery, to be used on its own or in conjunction with a neural
         // search
-        const subquery = {
-          multi_match: {
-            query,
-            type: 'most_fields',
-            fields: ['title^20', 'text'],
-            minimum_should_match: (isUrlSearch) ? 1 : '60%'
+        const subquery = [
+          {
+            multi_match: {
+              query,
+              type: 'most_fields',
+              fields: ['title^10', 'text'],
+              fuzziness: 'AUTO',
+              minimum_should_match: 1
+            }
+          },
+          // all match matches higher
+          {
+            multi_match: {
+              query,
+              type: 'most_fields',
+              fields: ['title^10', 'text'],
+              minimum_should_match: '100%',
+              boost: 100
+            }
+          },
+          // phrase match matches higher
+          {
+            multi_match: {
+              query,
+              type: 'phrase',
+              fields: ['title^10', 'text'],
+              boost: 1000
+            }
           }
-        }
+        ]
```

Comment:
This is the main thing that I added, which just enhances relevance for zaprank and other sorts by adding boosted exact matches.